docs/commands/mftf.md

```diff
@@ -463,18 +463,39 @@ The example parameters are taken from the `etc/config/.env.example` file.
 
 ### `static-checks`
 
-Runs all MFTF static-checks on the test codebase that MFTF is currently attached to.
+Runs all or specific MFTF static-checks on the test codebase that MFTF is currently attached to. 
+If no script name argument is specified, all existing static check scripts will run.
 
-#### Existing static checks
+#### Usage
 
-* Test Dependency: Checks that test dependencies do not violate Magento module's composer dependencies.
+```bash
+vendor/bin/mftf static-checks [<names>]...
+```
 
-#### Usage
+#### Examples
+
+To check what existing static check scripts are available
+
+```bash
+vendor/bin/mftf static-checks --help
+```
+
+To run all existing static check scripts 
 
 ```bash
 vendor/bin/mftf static-checks
 ```
 
+To run specific static check scripts
+
+```bash
+vendor/bin/mftf static-checks testDependencies
```

Comment:
When I run this command I get:

```
bin/mftf static-checks testDependencies

Invalid static check script(s): testDependencies. Please fix input arguments and rerun.
```

Reply:
Good catch! Fixed.

Reply:
I can pass a single value now.

But when I try to pass two values it says the second one doesn't exist:

```
$ bin/mftf static-checks reece testDependencies

Invalid static check script(s): testDependencies. Please fix input arguments and rerun.
```

Try to pass two values and it does not work.

Reply:
Fixed.